### ISSUE TITLE
Hover Scale not working

### ISSUE THREAD

**Issue by paiementstephanie** [paiementstephanie/infographic-effects-interactivity#1]
Hi Thomas, 

I'm just trying to get the icons in the cost section to get bigger in scale when you hover over them.  I already targeted them and also tried to do transform origin to them but nothing happens when I hover over the icons.  I'm not sure how to proceed.

**Comment by paiementstephanie** [comment]
@thomasjbradley 

I just realized I never tagged you above. _sigh_

**Comment by thomasjbradley** [comment]
There can't be any spaces between the word `scale` and the brackets—it should read like this:

``` css
scale(1.7, 1.7);
```

**Comment by paiementstephanie** [comment]
Thanks Thomas. So it's working now, however I'm seeing that it's not scaling it proportionately, and is only stretching it vertically.  Are my values not right?

**Comment by thomasjbradley** [comment]
Yeah, the second number defines the vertical scale, and the first number defines the horizontal scale. So, they both need to be the same if you want proportional scaling.

**Comment by paiementstephanie** [comment]
Perfect.  And I also noticed that on the smaller screens, and phones, that my waypoints don't go, because there's not enough scrolling space.  Do I do a media query somehow to make the scrolling points percent different on smaller screens? 
<img width="1440" alt="screen shot 2016-04-28 at 6 51 30 pm" src="https://cloud.githubusercontent.com/assets/10467614/14903883/69f9d960-0d72-11e6-9982-a42fa573291d.png">

**Comment by thomasjbradley** [comment]
You could try changing the percentange to exactly `bottom-in-view` and see if that helps.
